compiler/res/prog8lib/c64/textio.p8

```diff
@@ -468,6 +468,14 @@ asmsub  input_chars  (uword buffer @ AY) clobbers(A) -> ubyte @ Y  {
 	}}
 }
 
+; ---- screen , color address
+%asm {{
+    .weak
+    screen_addr   =   $0400
+    .endweak
```

Comment:
should this .endweak not be after color_addr?

Reply:
the color memory of the commodore 64 is in $ d800 cannot be changed